### PR TITLE
Added Animated Nav, Gallery(s), Quiz Apps, Landing Pages, Login Forms and Deleted Testimonials Section

### PR DESCRIPTION
**Pull request by Programing-School**
I deleted the testimonials section because it is not perfect

### CODE REVIEW

**Comment by Programing-School** [comment]
Who is Reviewing ?

**Comment by Programing-School** [comment]
# Please Accept

**Comment by Programing-School** [comment]
Hey Please Accept

File: Profile-Card-2/index.html [hunk 1]
```diff
@@ -1,38 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
```
Comment (mudit023) [inline Profile-Card-2/index.html:3]: Why are you making changes in someone else file, without creating an issue? If you want to improve someone else code first create an issue about it [here](https://github.com/mudit023/templates/issues) and then work on it if it gets assigned to you.

File: Profile-Card-2/opensource.css [hunk 2]
```diff
@@ -1,128 +1,131 @@
-
```
Comment (mudit023) [inline Profile-Card-2/opensource.css:1]: Don't make changes in other contributions if there is no issue created about it.

**Review by mudit023 — COMMENTED** [review]
I have told you multiple times don't make changes in multiple files from different contributors in the same Pull Request. Only contribute your template.

**Comment by mudit023** [comment]
Contribute your template and don't make changes in other contributions without creating an [issue](https://github.com/mudit023/templates/issues) about it.

**Comment by mudit023** [comment]
Close this PR and make a new one with just your template and no other changes.

File: Profile-Card-2/index.html (hunk 1 above)
Comment (Programing-School) [reply Profile-Card-2/index.html:3]: I am sorry

Comment (Programing-School) [reply Profile-Card-2/index.html:3]: But I created one pull request with multiple commits

You can accept one

File: Profile-Card-2/opensource.css (hunk 2 above)
Comment (Programing-School) [reply Profile-Card-2/opensource.css:1]: Ok

**Pull request closed without merging**